lib/pages/chat/chat_app_bar_title_style.dart

```diff
@@ -12,4 +12,7 @@ class ChatAppBarTitleStyle {
 
   static double get letterSpacingRoomName => 0.15;
   static double get letterSpacingStatusContent => 0.5;
+
+  static Color get flashingCircleBrightColor => const Color(0xFFcad1d9);
```

Comment:
it is 
<img width="244" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/6462404/c4d8527d-a14d-4352-a066-665187230d2e">

We also defined it in LinagoraSysColor. Please use it
```dart
LinagoraSysColors.material().tertiary
```
or
```dart
LinagoraSysColors.material().tertiaryDark
```